flink-core/src/main/java/org/apache/flink/configuration/Configuration.java

```diff
@@ -54,8 +54,12 @@
 	
 
 	/** Stores the concrete key/value pairs of this configuration object. */
-	private final HashMap<String, Object> confData;
-	
+	private final Map<String, Object> confData;
+
+	/** Stores the program version */
+	public final String FLINK_VERSION = getClass().getPackage().getImplementationVersion() ==
+		null ? "FLINK_TEST_VERSION" : getClass().getPackage().getImplementationVersion();
```

Comment:
What's the value of `FLINK_VERSION` if you execute a Flink job on a standalone cluster from your IDE using the `RemoteExecutionEnvironment`? Will it work?